arches/settings.py

```diff
@@ -519,6 +519,9 @@
 
 OAUTH2_PROVIDER = {"ACCESS_TOKEN_EXPIRE_SECONDS": 604800}  # one week
 
+TILE_EXCEL_EXPORT_TEMP_DIRECTORY = None
+TILE_EXCEL_EXPORT_TEMP_FILE_DELETE = True
+
```

Comment:
I'm surprised the temp files weren't cleaned up--is that's a Python bug that's been fixed in later versions? If so, then we can avoid increasing the surface area of the settings here a bit. Depending on the outcome of the prior comment, by adding `os.unlink(tmp.name)` we may not need to specify a directory at all, but let me know if I'm not grokking your use case! 🙏 

If we do keep the directory setting, it might be nice to use it on all platforms.